lib/pengine/pe_actions.c

```diff
@@ -714,13 +714,27 @@ pcmk__unpack_action_meta(pcmk_resource_t *rsc, const pcmk_node_t *node,
 
     meta = pcmk__strkey_table(free, free);
 
-    // Cluster-wide <op_defaults> <meta_attributes>
-    pe__unpack_dataset_nvpairs(rsc->priv->scheduler->priv->op_defaults,
-                               PCMK_XE_META_ATTRIBUTES, &rule_data, meta, NULL,
-                               FALSE, rsc->priv->scheduler);
+    if (action_config != NULL) {
```

Comment:
Call this out as a Fix: rather than Low:, and mention in the commit message that this brings it into line with how option blocks everywhere else behave